### DISCUSSION TITLE
"Fake" param?

### DISCUSSION THREAD

**Discussion by nhorton** [avo-hq/avo#2013] [Help]
In an action, I have a date field called `end_date`. When the action posts, I am getting an unpermitted param warning for `fake_end_date` which appears nowhere in my code.

Any idea what that is and if I can get rid of it?

**Comment by adrianthedev** [comment]
We are using a fack input to hold state, and that input value is being sent to the server, but ignored.
Is that a bother to you? Are you getting alerts from any service or gems that that is there?
